client/my-sites/checkout/src/components/vat-form/style.css

```diff
@@ -16,3 +16,7 @@
 .vat-form__row input[type="checkbox"]:disabled {
 	opacity: 0.5;
 }
+
+.is-section-checkout .components-checkbox-control__input[type=checkbox]:checked {
```

Comment:
It's failing the code style check by the missing quote, we might want to update this to `type="checkbox"`?